ios/FluentUI/Button/Button.swift

```diff
@@ -130,6 +130,11 @@ open class Button: UIButton, Shadowable, TokenizedControlInternal {
         tokenSet.registerOnUpdate(for: self) { [weak self] in
             self?.update()
         }
+
+        NotificationCenter.default.addObserver(self,
```

Comment:
Does this mean that all UIKit-based controls need to observe this notification again? We recently removed this to move it into `ControlTokenSet` to reduce duplicated boiler-plate.

Reply:
Yeah, it would mean that we'd have to move it back to boilerplate code on all the UIKit controls.

Reply:
But, we still have to add `willMove(toWindow: )` to all of the UIKit controls, and we need different theme management systems between UIKit and SwiftUI, so having the observer in the actual control instead of the shared token set makes sense.